data/mongodb/v0/config/definition.json

```diff
@@ -0,0 +1,26 @@
+{
+  "availableTasks": [
+    "TASK_INSERT",
+    "TASK_FIND",
+    "TASK_UPDATE",
+    "TASK_DELETE",
+    "TASK_DROP_COLLECTION",
+    "TASK_DROP_DATABASE"
```

Comment:
question
What use case or design for to add  `"TASK_DROP_COLLECTION",    "TASK_DROP_DATABASE"`

Reply:
And, if we can drop collection & database, how about we also can create collection & database?

Reply:
@chuang8511

I'm thinking of when the user wants to migrate from one database to another, it could be the whole database or just some specific collection, and when the process has finished, they may want to delete the old data source so that might be the use case.

As for create collection & database, `insert` method from mongodb sdk provide an auto create feature if either collection or database doesn't exist. Moreover, mongodb have flexible structure unlike sql, in a single collection, it may have various fields for each documents. So I guess there's no need to add create collection and database.

Reply:
> they may want to delete the old data source so that might be the use case.

Seems good in terms of cost.

> insert method from mongodb sdk provide an auto create feature if either collection or database doesn't exist.

OK! Good!